README.md

```diff
@@ -41,6 +41,9 @@ akka.persistence{
 		
 			# metadata table
 			metadata-table-name = metadata
+
+			# defines column db type used to store payload. Available option: BYTEA (default), JSON, JSONB
+			stored-as = BYTEA
```

Comment:
Do you think it is a good idea to have many schemas in the one plugin? How will we support it? We could choose json and live with it, Anyway we haven't released any version since January. We have zero users of this plugin, so we won't have any real breaking changes